script.sh

```diff
@@ -63,7 +63,6 @@ fi
 
 Info "Starting build of ${SOURCE} using travis.debian.net"
 
-TRAVIS_DEBIAN_MIRROR="${TRAVIS_DEBIAN_MIRROR:-http://deb.debian.org/debian}"
```

Comment:
?

Reply:
Yes, as we are autodetecting and setting TRAVIS_DEBIAN_MIRROR based on the derivative (or lack of) name, we are setting the "Debian" case there.
This enable us still setting a derivative and overriding TRAVIS_DEBIAN_MIRROR (as empty == not set)

Reply:
Thanks, I thought for a second this would make it not overridable. (can see that is not the case)